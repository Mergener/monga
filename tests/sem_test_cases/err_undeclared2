function foo() {
    i = 5;
    var i : int;
    return i;
}