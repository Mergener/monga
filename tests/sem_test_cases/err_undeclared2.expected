[Error] At line 2, column 8: Variable 'i' was not declared in this scope.
[Error] At line 4, column 14: Cannot return value of type 'int' in function declared with return type 'void'.
Semantic errors were caught during analysis.
